### ISSUE TITLE
Transmutation table not showing items

### ISSUE THREAD

**Issue by Killerhurtz** [sinkillerj/ProjectE#1787]
Exact ProjectE version:  1.12-PE1.3.2h (Upgraded from 1.12-PE1.3.1B)

Exact Forge version: 14.23.5.2772

Link to crash log : No crash

Steps to reproduce:

1. Start a new world with a significant modlist (modlist available on request. ProjectE related mods: ProjectE, Equivalent Additions 2r, Equivalent Integrations PE1.3.1B)
2. Play for some amount of time, closing and reopening the game over the course of a few days

What I expected to happen: All of the items I previously learned are available and visible

What happened instead: Several items, most of which above 32 EMC, are not displayed - and adding them back to the table has the behavior as if the table already knew the item.

**Comment by bobthebigbat** [comment]
Happened to me too. Playing Stoneblock 2 v1.2.0 on a dedicated server.
Most EMC and learned items gone after changing dimensions/worlds.
Might have revered to a version of the table from an hour or so earlier before changing dimensions.
Relogging in didn't fix it this time.

**Comment by bobthebigbat** [comment]
> Do you have an item in the center slot of the transmutation table?
> 
> This slot acts as a filter only showing items with less emc than the provided item.

No. I had no item in the middle slot. Even if there was it wouldn't account for the 200,000,000,000+ EMC that disappeared also. The issue was a result of changing dimensions (for me anyhow).
